ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -741,10 +741,46 @@ export class ObjectDetailComponent implements OnInit {
       srcDesc: undefined,
       srcOrder: '',
     })
-
     this.setIndexRows()
   }
 
+  restoreSpannerTable() {
+    this.data
+      .restoreTable(this.currentObject!.name)
+      .pipe(take(1))
+      .subscribe((res: string) => {
+        if (res === '') {
+          this.isObjectSelected = false
+        }
+      })
+    this.currentObject = null
+  }
+
+  dropTable() {
+    let openDialog = this.dialog.open(DropIndexOrTableDialogComponent, {
+      width: '35vw',
+      minWidth: '450px',
+      maxWidth: '600px',
+      data: { name: this.currentObject?.name, type: 'Table' },
```

Comment:
Better to define these type values - Table and Index in constants to avoid errors in case matching

Reply:
Updated